composer.json

```
@@ -1,6 +1,6 @@
 {
-    "name": "ozee31/cakephp-cors",
-    "description": "A CakePHP (3.3.x) plugin for activate cors domain in your application",
+    "name": "rabp99/cakephp-cors",
```

Please, keep the original name here.
```suggestion
    "name": "ozee31/cakephp-cors",
```